projects/components/table/src/subcomponents/table-actions.component.ts

```diff
@@ -20,4 +21,8 @@ export class PsTableActionsComponent {
   @Output() public showSettings = new EventEmitter<void>();
 
   @ViewChild('menu', { static: true }) menu: MatMenu;
+
+  public get showIcon(): boolean {
```

Comment:
This should be calculated once in an OnInit or OnChanges method. This way it will be calculated three times every time the changeDetections deems it neccessary.

Reply:
seems reasonable